### ISSUE TITLE
I found such a strange thing. Add code anywhere in the definition model that is not associated with the current model, such as torch.nn.Linear () will affect the model

### ISSUE THREAD

**Issue by tanghui2017108** [pytorch/pytorch#47835]
### Please note that this issue tracker is not a help form and this issue will be closed.

We have a set of [listed resources available on the website](https://pytorch.org/resources). Our primary means of support is our discussion forum:

- [Discussion Forum](https://discuss.pytorch.org/)

**Comment by tanghui2017108** [comment]
![Uploading image.png…]()

**Comment by tanghui2017108** [comment]
![image](https://user-images.githubusercontent.com/32611793/98908031-bb6f7580-24fa-11eb-9c8e-c71c20925039.png)

**Comment by ptrblck** [comment]
Each layer creation will randomly initialize the parameters (if available) and will thus call into the pseudorandom number generator. If your training is sensitive to different seeds, these additional layer creations can change the training results significantly and you should see the same effect by removing the layer creation and just changing the seed at the beginning of your script.

PS: the forum is a better place for questions like that.

**Comment by tanghui2017108** [comment]
> Each layer creation will randomly initialize the parameters (if available) and will thus call into the pseudorandom number generator. If your training is sensitive to different seeds, these additional layer creations can change the training results significantly and you should see the same effect by removing the layer creation and just changing the seed at the beginning of your script.
> 
> PS: the forum is a better place for questions like that.

Your answer is very good, thanks.
